app/assets/javascripts/backbone/views/timeline_table_view.coffee

```diff
@@ -177,3 +183,12 @@ class MS.Views.TimelineTableView extends Backbone.View
     $topGroups.removeClass('dropup')
     $topSteps = $topRows.find('.step-menu-col')
     $topSteps.removeClass('dropup')
+
+  highlightStep: (e) ->
+    if !(@$(e.target).hasClass('step-start-date') || @$(e.target).hasClass('step-end-date'))
```

Comment:
Only specific step start dates and all step end dates have the dependent/precedent relationship.

The check would could use is to see if a `.step-start-date` element has a value for `data-precedent-id`. And we can check if a `.step-end-date` element has a value for `data-precedent-id` or `data-dependent-step-start-date`.